src/js/components/Select/SelectContainer.js

```diff
@@ -277,6 +277,58 @@ const SelectContainer = forwardRef(
       [activeIndex, isDisabled],
     );
 
+    const findNextActiveIndex = (
+      event,
+      selectOptions,
+      currentActiveIndex,
```

Comment:
I think we can simplify this to not pass currentActiveIndex and afterActiveIndex. Looking at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select, adding another option for Crocodile after Hamster, typing H selects H Hamster, then typing C goes back to Cat instead of down to Crocodile.

Reply:
Thanks for the feedback, Eric. Will simplify. I was attempting to make it easier to iterate across options in the case of multiple options with the same first letter (Car, Cat, Caterpillar, ...). That said, makes sense to follow the HTML Select spec as close as possible.